src/lang/english.txt

```diff
@@ -194,6 +194,7 @@ STR_COLOUR_DEFAULT                                              :Default
 STR_UNITS_VELOCITY_IMPERIAL                                     :{COMMA}{NBSP}mph
 STR_UNITS_VELOCITY_METRIC                                       :{COMMA}{NBSP}km/h
 STR_UNITS_VELOCITY_SI                                           :{COMMA}{NBSP}m/s
+STR_UNITS_VELOCITY_PLAYORIENTED                                 :{COMMA}{NBSP}decitiles/day
```

Comment:
This will make several strings much longer, potentially "breaking" the UI, maybe shorten it to `dt/d`?

Reply:
nobody will understand this unit

Reply:
Also, I don't think it's a big problem if the string "decitiles/day" itself gets truncated; only if the length of that string causes stuff _following_ the "100 mph" string to be truncated.

We might get away with "dt/d" (or "deci-tpd" or "tiles/10d" or whatever) if we also put it in a parenthetical in the drop-down menu in the settings dialog where people choose this unit in the first place, something like `Play-oriented (tiles per 10 days: dt/d)`. But these wouldn't be nearly as self-explanatory as "decitiles/day" is.

Reply:
This is also somewhat mitigated by the change from "decitiles/day" to "tiles/day", although that's admittedly still longer than "mph" and "km/h".